cloud_edition/flexibility_mode/docs/disk_usage.rst

```diff
@@ -0,0 +1,112 @@
+    
+    echo "Track assets that will be deleted in a csv file"
+    mysql akeneo_pim -e "SELECT fi.id, fi.original_filename, fi.file_key, r.file_info_id, r.asset_id, v.* FROM akeneo_file_storage_file_info fi LEFT JOIN pimee_product_asset_reference r ON fi.id = r.file_info_id LEFT JOIN pimee_product_asset_variation v ON fi.id = v.file_info_id WHERE storage = 'assetStorage' AND r.file_info_id IS NULL AND r.asset_id IS NULL AND v.source_file_info_id IS NULL" > /home/akeneo/purge/asset_rows_to_delete.csv
+  
+    echo "List files to be deleted in a txt file"
+    echo "Path of files to be deleted is calculated by concatenating "/home/akeneo/pim/app/file_storage/asset/" with value of "fi.file_key" from the MySQL resquest."
+    mysql akeneo_pim -se "SELECT concat('/home/akeneo/pim/app/file_storage/asset/',fi.file_key) FROM akeneo_file_storage_file_info fi LEFT JOIN pimee_product_asset_reference r ON fi.id = r.file_info_id LEFT JOIN pimee_product_asset_variation v ON fi.id = v.file_info_id WHERE storage = 'assetStorage' AND r.file_info_id IS NULL AND r.asset_id IS NULL AND v.source_file_info_id IS NULL" > /home/akeneo/purge/asset_files_to_delete.txt
+  
+    echo "Delete rows in database"
+    mysql akeneo_pim -se "SET FOREIGN_KEY_CHECKS=0;DELETE v, r, fi FROM akeneo_file_storage_file_info fi LEFT JOIN pimee_product_asset_reference r ON fi.id = r.file_info_id LEFT JOIN pimee_product_asset_variation v ON fi.id = v.file_info_id WHERE storage = 'assetStorage' AND r.file_info_id IS NULL AND r.asset_id IS NULL AND v.source_file_info_id IS NULL;SET FOREIGN_KEY_CHECKS=1;" > /home/akeneo/purge/asset_rows_deleted.csv
+  
+    echo "Delete assets files based on previous list"
+    cat /home/akeneo/purge/asset_files_to_delete.txt | xargs rm -f
+    
+
+- purge old versions from versioning table
+
+.. code-block:: bash
+
+    echo "Check pim_versioning_version table size"
+    mysql akeneo_pim -e 'SELECT table_name AS `Table`, round(((data_length + index_length) / 1024 / 1024 / 1024), 2) `Size in GB` FROM information_schema.TABLES WHERE table_schema = "akeneo_pim" AND table_name = "pim_versioning_version";'
+  
+
+
+.. warning::
+
+    **Warning:** `mysqlcheck --optimize` will duplicate the table(s) before optimizing it (them). Which means that, before running the command, one must make sure that the remaining free space is at least equals to the biggest table size. To avoid any data loss, backing the tables up before running `mysqlcheck` is prefered. For more information: https://dev.mysql.com/doc/refman/5.7/en/mysqlcheck.html
+
+
+.. warning::
+
+    **Warning:** `mysqlcheck --optimize` will lock the table during the operation. Hence the table will be unavailable for the PIM. For more information: https://dev.mysql.com/doc/refman/5.7/en/mysqlcheck.html
+
+
+.. code-block:: bash
+
+    mkdir -p /home/akeneo/purge
+    
+    echo "Cleansing versions older than 90 days"
+    nohup php bin/console --env=prod pim:versioning:purge --more-than-days 90 --force -n &
```

Comment:
I would not suggest here the use of `nohup` nor `&` because it is the choice of the integrator.